.eslintrc

```diff
@@ -90,10 +90,10 @@
     "no-use-before-define": 2,
     "radix": 2,
     "curly": 2,
-    "no-fallthrough": 0,
-    "default-case": 0,
-    "no-plusplus": 0,
-    "dot-notation": 0
+    "no-fallthrough": 2,
+    "default-case": 2,
+    "no-plusplus": 2,
```

Comment:
PlusPlus (and MinusMinus) can be very tempting to use in weird compound statements, like

`doMyThing(i++)`

Which is why people tend to ban it (and newer languages tend to omit it)